src/test/java/teammates/test/cases/logic/FeedbackSessionsLogicTest.java

```diff
@@ -2019,7 +2037,7 @@ public void testGetSessionResultsForUser_instructorAllQuestions_shouldGenerateCo
                 getResponseId("qn5.resp1", responseBundle) + "={false,true}",
                 getResponseId("qn6.resp1", responseBundle) + "={true,true}");
         AssertHelper.assertContains(expectedStrings, mapString);
-        assertEquals(10, bundle.getResponseVisibilityTable().size());
+        assertEquals(58, bundle.getResponseVisibilityTable().size());
```

Comment:
58 = 10 (total response) + 48 (total missing). It's not easy to justify where 10 and 48 come from, but 58 should not be left as a magic number.